src/tlog/verify.ts

```diff
@@ -31,12 +34,11 @@ import {
 // key.
 export function verifyTLogSET(
   bundle: Bundle,
-  publicKey: string,
```

Comment:
Note that we no longer need to pass-in the signing key since we don't have to re-create the original Rekor entry.